packages/ui-components/package.json

```diff
@@ -0,0 +1,108 @@
+{
+  "name": "ui-components",
+  "version": "0.0.1",
+  "description": "Reusable UI Components for Agoric Dapps, built with React and MaterialUI",
+  "main": "dist/index.js",
```

Comment:
I tried using the `src` directly, and it doesn't work in dapp-token-economy. I get an error message telling me to change my Babel settings to allow for JSX, but since we haven't ejected, I haven't found an easy way to change the Babel settings :/ There must be a way to bundle it correctly, though, since we can import MaterialUI components from packages.